app-client/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM adoptopenjdk/openjdk11-openj9:jdk-11.0.1.13-alpine-slim
+COPY build/libs/app-client-*.jar app-client.jar
+EXPOSE 8081
+CMD java -jar app-client.jar
```

Comment:
Haven't deleted it yet because I get a **"gradle assembler"** not found error when I try building the **cdc-client** with the Dockerfile in 

> src/main/docker/Dockerfile